Formula/julia-binary.rb

```diff
@@ -1,10 +1,10 @@
 class JuliaBinary < Formula
   desc "The Julia programming language"
   homepage "https://julialang.org"
-  version "0.6.1"
+  version "1.5"
 
-  url "https://julialang-s3.julialang.org/bin/linux/x64/0.6/julia-0.6.1-linux-x86_64.tar.gz"
-  sha256 "d73f988b4d5889b30063f40c2f9ad4a2487f0ea87d6aa0b8ed53e789782bb323"
+  url "https://mirrors.bfsu.edu.cn/julia-releases/bin/linux/x64/1.5/julia-1.5-latest-linux-x86_64.tar.gz"
```

Comment:
Since you use a mirror URL here, let's add an option, so that users can decide to either download julia binary from official site or from a mirror site:

```ruby
class JuliaBinary < Formula
    ...

    option "with-bfsu-mirror", "Download from https://mirrors.bfsu.edu.cn, use this option if you're in mainland China for a faster downloading speed."

    if build.with? "bfsu-mirror"
        url "https://mirrors.bfsu.edu.cn/julia-releases/bin/linux/x64/1.4/julia-1.4.2-linux-x86_64.tar.gz"
    else
        url "https://julialang-s3.julialang.org/bin/linux/x64/1.4/julia-1.4.2-linux-x86_64.tar.gz"
    end

    sha256 "xxx"

    ...
end
```

To use that option, issue `brew install julia-binary --with-bfsu-mirror`.

Reply:
Than you! This is greatly helpful!